[2022-03-20 20:55:46,088] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: fetch_news.prepare_insert 2022-03-01T00:00:00+00:00 [queued]>
[2022-03-20 20:55:46,140] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: fetch_news.prepare_insert 2022-03-01T00:00:00+00:00 [queued]>
[2022-03-20 20:55:46,145] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2022-03-20 20:55:46,146] {taskinstance.py:1043} INFO - Starting attempt 3 of 3
[2022-03-20 20:55:46,146] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2022-03-20 20:55:46,193] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): prepare_insert> on 2022-03-01T00:00:00+00:00
[2022-03-20 20:55:46,216] {standard_task_runner.py:52} INFO - Started process 499 to run task
[2022-03-20 20:55:46,239] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'fetch_news', 'prepare_insert', '2022-03-01T00:00:00+00:00', '--job-id', '260', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/fetch_news.py', '--cfg-path', '/tmp/tmpr_1mf1ai', '--error-file', '/tmp/tmpltj9jm0a']
[2022-03-20 20:55:46,239] {standard_task_runner.py:77} INFO - Job 260: Subtask prepare_insert
[2022-03-20 20:55:46,416] {logging_mixin.py:104} INFO - Running <TaskInstance: fetch_news.prepare_insert 2022-03-01T00:00:00+00:00 [running]> on host e39215f0be5b
[2022-03-20 20:55:46,687] {taskinstance.py:1257} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=fetch_news
AIRFLOW_CTX_TASK_ID=prepare_insert
AIRFLOW_CTX_EXECUTION_DATE=2022-03-01T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-03-01T00:00:00+00:00
[2022-03-20 20:55:46,700] {python.py:118} INFO - Done. Returned value was: None
[2022-03-20 20:55:46,793] {taskinstance.py:1166} INFO - Marking task as SUCCESS. dag_id=fetch_news, task_id=prepare_insert, execution_date=20220301T000000, start_date=20220320T205546, end_date=20220320T205546
[2022-03-20 20:55:47,120] {taskinstance.py:1220} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2022-03-20 20:55:47,180] {local_task_job.py:146} INFO - Task exited with return code 0
